### PR TITLE
fix Windows build libmygpo-qt5

### PR DESCRIPTION
**Pull request by pavelvat**
libmygpo-qt5 is static library.
Clementine/3rdparty/libmygpo-qt5/src/CMakeLists.txt:
```
add_library( ${MYGPO_QT_TARGET_NAME} STATIC ${LIBMYGPO_QT_SRC} ${LIBMYGPO_QT_MOC_SRC} )
```

log: https://gist.github.com/pavelvat/2072ae11fb8c94fba317c371eaeba871

### CODE REVIEW

File: 3rdparty/libmygpo-qt5/src/mygpo_export.h [hunk 1]
```diff
@@ -25,6 +25,8 @@
 
 #include <QtCore/qglobal.h>
 
+#define MYGPO_EXPORT
```
Comment (hatstand) [inline 3rdparty/libmygpo-qt5/src/mygpo_export.h]: This is pretty weird with the `#ifndef MYGPO_EXPORT` below

Comment (pavelvat) [inline 3rdparty/libmygpo-qt5/src/mygpo_export.h]: What you propose?

Something like this:
```
#ifndef MYGPO_BUILD_TYPE_STATIC
# if defined(MYGPO_MAKEDLL)
/* We are building this library */
#  define MYGPO_EXPORT Q_DECL_EXPORT
# else
/* We are using this library */
#  define MYGPO_EXPORT Q_DECL_IMPORT
# endif
#else
# define MYGPO_EXPORT
#endif
```
?

Comment (hatstand) [inline 3rdparty/libmygpo-qt5/src/mygpo_export.h]: LGTM

**Comment by pavelvat** [comment]
Done.

**Pull request merged**